language/compiler/ir-to-bytecode/syntax/src/syntax.rs

```diff
@@ -1969,12 +1975,12 @@ fn parse_script_or_module<'input>(
     }
 }
 
-pub fn parse_cmd_string<'input>(
+pub fn parse_cmd_string_<'input>(
```

Comment:
This one doesn't make sense to me. The "_string" suffix is intended to designate that this function takes a string input (vs. the other "parse_cmd_" function). We shouldn't put the underscore after that suffix. If anything, this should be "parse_cmd__string" but personally I would just ignore that inconsistency and leave the name as it is.

I'll approve this so you can go ahead after addressing that.

Reply:
What about `parse_string_cmd_`?

Reply:
thanks!